proto/decentraland/sdk/components/avatar_customization.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+package decentraland.sdk.components;
+import "decentraland/common/colors.proto";
+import "decentraland/sdk/components/common/id.proto";
+option (common.ecs_component_id) = 1087;
+
+// AvatarCustomizations sets all modifiers over the avatar's apparence. This 
+message PBAvatarCustomization {
+  decentraland.common.Color4 skin_color = 1;
```

Comment:
the `PBAvatarShape` and [AvatarInfo schema](https://github.com/decentraland/schemas/blob/a5197eed1108df78845bc3f364c1c275a2bf1403/src/platform/profile/avatar.ts#L33) use `Color3`s - i think these should match